diff.py

```diff
@@ -1961,7 +1963,29 @@ def process(dump: str, config: Config) -> List[Line]:
             row_parts = [part.lstrip() for part in row.split(" ", 1)]
         mnemonic = row_parts[0].strip()
         args = row_parts[1] if len(row_parts) >= 2 else ""
-        row = mnemonic + "\t" + args.replace("\t", "  ")
```

Comment:
this should still happen, even for non-ppc

Reply:
Since this happens regardless of arch, move it to after `pre_process`

Reply:
sounds good, before it was setup where it needed to happen inside of pre_process, but after lots of refactoring and the change you suggested about moving li-->addi to only in sda relocs.

Okay, but to move this like to after pre_process, it means preprocess will now need to return a tuple of updated mnemonic and args.